components/sysutils/lsof/Makefile

```diff
@@ -30,6 +31,9 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/justmake.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+# Need GNU version of yes
+PATH = $(PATH.gnu)
+
```

Comment:
Okay, I've done that.  The comment now reads:

# In oi_151a8, illumos version of yes ignores SIGPIPE, never exits